### DISCUSSION TITLE
Why isn't there a register function anymore?

### DISCUSSION THREAD

**Discussion by jberculo** [Jacobtims/filament-breezy#239] [General]
Hi, I'm just wondering why it has been decided to not include a register option anymore in the package? Can't really find an explanation for the differences between v1 and v2

Thanks!

**Comment by joshembling** [comment]
Can't see a reset / forgotten password function anymore either 

Looks like everything that was in `\JeffGreco13\FilamentBreezy\Http` has either moved or been deleted/replaced.

**Comment by DSpeichert** [comment]
It's because Filament v3 now natively supports:
* login
* registration
* forgotten password

Somehow it's not very well documented but check these methods out in your Panel provider:
```php
return $panel
    ->default()
    ->id('app')
    ->path('panel')
    ->login()
    ->emailVerification()
    ->passwordReset()
    ->registration()
...
```

**Reply by jberculo** [reply]
Oh thanks! I was hoping for so much, but couldnt find it in the docs!

**Reply by retifex** [reply]
Thanks @DSpeichert. Just wanted to add: The [documentation ](https://filamentphp.com/docs/3.x/panels/users#authentication-features)says, the method name is `emailVerification()`. As far as i can see, requiresEmailVerification() is just a helper method.
